src/main/java/io/github/thebusybiscuit/slimefun4/implementation/tasks/RadioactivityTask.java

```diff
@@ -0,0 +1,100 @@
+package io.github.thebusybiscuit.slimefun4.implementation.tasks;
+
+import io.github.thebusybiscuit.cscorelib2.chat.ChatColors;
+import io.github.thebusybiscuit.slimefun4.api.player.PlayerProfile;
+import io.github.thebusybiscuit.slimefun4.api.player.StatusEffect;
+import io.github.thebusybiscuit.slimefun4.core.attributes.ProtectionType;
+import io.github.thebusybiscuit.slimefun4.core.attributes.Radioactive;
+import io.github.thebusybiscuit.slimefun4.implementation.SlimefunPlugin;
+import io.github.thebusybiscuit.slimefun4.implementation.items.RadioactiveItem;
+import io.github.thebusybiscuit.slimefun4.utils.itemstack.RadiationSymptom;
+import me.mrCookieSlime.Slimefun.Objects.SlimefunItem.SlimefunItem;
+import net.md_5.bungee.api.ChatMessageType;
+import net.md_5.bungee.api.chat.ComponentBuilder;
+import org.bukkit.Bukkit;
+import org.bukkit.GameMode;
+import org.bukkit.NamespacedKey;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.potion.PotionEffect;
+import org.bukkit.potion.PotionEffectType;
+
+import javax.annotation.Nonnull;
+
+/**
+ * The {@link RadioactivityTask} handles radioactivity for
+ * {@link Radioactive} items.
+ *
+ * @author Semisol
+ *
+ */
+public class RadioactivityTask implements Runnable {
+
+    private static final RadiationSymptom[] SYMPTOMS = RadiationSymptom.values();
+    private static StatusEffect RADIATION_EFFECT;
+
+    {
+        assert SlimefunPlugin.instance() != null;
+        RADIATION_EFFECT = new StatusEffect(new NamespacedKey(SlimefunPlugin.instance(), "radiation"));
+    }
+
+    public static void removePlayer(@Nonnull Player p) {
+        RADIATION_EFFECT.clear(p);
+    }
```

Comment:
Not sure if this task is the right place for this.

I think a proper `RadiationUtils` class is better here, this way the API can be much more open, something like

`RadiationUtils.addExposure(Player, exposureLevel)`
`RadiationUtils.removeExposure(Player, exposureLevel)`
`RadiationUtils.resetExposure(Player)`